### DISCUSSION TITLE
Error message 'Process completed wit exit code 2' (validate secrets)

### DISCUSSION THREAD

**Discussion by Juli0787** [community/community#107446] [New to GitHub]
Hej Community, anyone who can help me solving the error message 'Process completed with exit code 2'. It appeared when throughout the validation of my secrets. Highly appreciate your support so that I can install the Loop 3 app.
Best and thanks, Julia

**Comment by queenofcorgis** [comment]
Hi Julia! 

Would you mind sharing your code so the community could better assist here?

**Reply by Juli0787** [reply]
Hej Lili,
Many thanks for your understanding, and apologize me just coming back to the community today! Please find the errors below:
60 Error: GH_PAT secret is set but invalid or lacking appropriate privileges on the Juli0787/Match-Secrets repository. Verify that it is set correctly and try again.
99 Error: Process completed with exit code 2.
Ich highly appreciate your support.
Best,
Julia

**Comment by koeppel** [comment]
Hi Julia,

it seems like that there is an issue with your GH_PAT secret while trying to build your Loop app. I'd suggest checking the following:

## 1. Does the **Match-Secrets** repository exist and is the name spelled correct?

## 2. Does the GH_PAT secret have the correct privileges? 
You can check this by clicking in the top right corner -> Settings -> Developer Settings -> Personal access tokens -> Tokens (classic) -> Click on GH-PAT -> Selected scopes 

Check if workflow and everything under the repo section is selected.

## Otherwise

- I'd suggest taking another look at [How to Build the Loop App With a Web Browser](https://youtu.be/kiu5ho0MTW8?si=_lF_-5fs3ElZuvo6&t=628 ) - with timestamp where they create the PAT
- [Open an issue in the Loop repository](https://github.com/LoopKit/Loop/issues/new/choose)

If this resolved your question please mark this response as answer so that other users with the same issue can find it directly :)
